### ISSUE TITLE
[🐛 Bug]: 

### ISSUE THREAD

**Issue by sriramsridharanvr** [SeleniumHQ/selenium#11811]
### What happened?

I'm unable to launch the Edge Browser with IE mode using the RemoteWebDriver, though i'm able to get that working using the local WebDriver.

I'm running the `selenium-server` on my remote machine (which at the moment is my localhost)

` java -jar .\selenium-server-4.8.1.jar standalone`

Now, I can go to `http://localhost:4444` and can see my server running.

In my maven project, I try to launch Edge with IE mode, using the below code:

```
public static void runIEWithEdge(String[] args) throws MalformedURLException {
        String remoteUrl = "http://localhost:4444";

        InternetExplorerOptions options = new InternetExplorerOptions();
        options.attachToEdgeChrome();

        // Create RemoteWebDriver with the EdgeOptions and remote URL
        WebDriver driver = new RemoteWebDriver(new URL(remoteUrl), options);

        // Navigate to a webpage
        driver.get("http://demo.borland.com/gmopost");

        // Do some automation tasks
        driver.findElement(By.name("bSubmit")).click();


        // Close the browser
        driver.quit();

    }
```

When it runs, I get an exception in the `selenium-server`, like below:

```
12:28:29.968 WARN [SeleniumSpanExporter$1.lambda$export$1] - org.openqa.selenium.SessionNotCreatedException: Could not start a new session. null
Host info: host: 'YCSLAPTEMP', ip: '192.168.16.118'
Build info: version: '4.8.1', revision: '8ebccac989'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: driver.version: unknown
        at org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:215)
        at org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:69)
        at org.openqa.selenium.grid.node.local.SessionSlot.apply(SessionSlot.java:147)
        at org.openqa.selenium.grid.node.local.LocalNode.newSession(LocalNode.java:384)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor.startSession(LocalDistributor.java:645)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor.newSession(LocalDistributor.java:564)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.handleNewSessionRequest(LocalDistributor.java:818)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.lambda$run$1(LocalDistributor.java:779)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)

12:28:29.970 WARN [SeleniumSpanExporter$1.lambda$export$3] - {"traceId": "3b7d83ab379547e1b109a4fb9bd94f14","eventTime": 1679641109965654800,"eventName": "exception","attributes": {"exception.message": "Unable to create session: Could not start a new session. null \nHost info: host: 'YCSLAPTEMP', ip: '192.168.16.118'\nBuild info: version: '4.8.1', revision: '8ebccac989'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'\nDriver info: driver.version: unknown","exception.stacktrace": "org.openqa.selenium.SessionNotCreatedException: Could not start a new session. null \nHost info: host: 'YCSLAPTEMP', ip: '192.168.16.118'\nBuild info: version: '4.8.1', revision: '8ebccac989'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'\nDriver info: driver.version: unknown\r\n\tat org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:215)\r\n\tat org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:69)\r\n\tat org.openqa.selenium.grid.node.local.SessionSlot.apply(SessionSlot.java:147)\r\n\tat org.openqa.selenium.grid.node.local.LocalNode.newSession(LocalNode.java:384)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor.startSession(LocalDistributor.java:645)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor.newSession(LocalDistributor.java:564)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.handleNewSessionRequest(LocalDistributor.java:818)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.lambda$run$1(LocalDistributor.java:779)\r\n\tat java.base\u002fjava.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base\u002fjava.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base\u002fjava.lang.Thread.run(Thread.java:833)\r\n","exception.type": "org.openqa.selenium.SessionNotCreatedException","logger": "org.openqa.selenium.grid.distributor.local.LocalDistributor","request.payload": "[Capabilities {browserName: internet explorer, se:ieOptions: {ie.edgechromium: true}}]"}}
```

### How can we reproduce the issue?

```shell
Run the selenium-server in standalone mode. Try to invoke a RemoteWebDriver that points to http://localhost:4444.
```

### Relevant log output

```shell
12:28:29.968 WARN [SeleniumSpanExporter$1.lambda$export$1] - Unable to create session: Could not start a new session. null
Host info: host: 'YCSLAPTEMP', ip: '192.168.16.118'
Build info: version: '4.8.1', revision: '8ebccac989'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: driver.version: unknown
12:28:29.968 WARN [SeleniumSpanExporter$1.lambda$export$1] - org.openqa.selenium.SessionNotCreatedException: Could not start a new session. null
Host info: host: 'YCSLAPTEMP', ip: '192.168.16.118'
Build info: version: '4.8.1', revision: '8ebccac989'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: driver.version: unknown
        at org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:215)
        at org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:69)
        at org.openqa.selenium.grid.node.local.SessionSlot.apply(SessionSlot.java:147)
        at org.openqa.selenium.grid.node.local.LocalNode.newSession(LocalNode.java:384)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor.startSession(LocalDistributor.java:645)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor.newSession(LocalDistributor.java:564)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.handleNewSessionRequest(LocalDistributor.java:818)
        at org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.lambda$run$1(LocalDistributor.java:779)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)

12:28:29.970 WARN [SeleniumSpanExporter$1.lambda$export$3] - {"traceId": "3b7d83ab379547e1b109a4fb9bd94f14","eventTime": 1679641109965654800,"eventName": "exception","attributes": {"exception.message": "Unable to create session: Could not start a new session. null \nHost info: host: 'YCSLAPTEMP', ip: '192.168.16.118'\nBuild info: version: '4.8.1', revision: '8ebccac989'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'\nDriver info: driver.version: unknown","exception.stacktrace": "org.openqa.selenium.SessionNotCreatedException: Could not start a new session. null \nHost info: host: 'YCSLAPTEMP', ip: '192.168.16.118'\nBuild info: version: '4.8.1', revision: '8ebccac989'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'\nDriver info: driver.version: unknown\r\n\tat org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:215)\r\n\tat org.openqa.selenium.grid.node.config.DriverServiceSessionFactory.apply(DriverServiceSessionFactory.java:69)\r\n\tat org.openqa.selenium.grid.node.local.SessionSlot.apply(SessionSlot.java:147)\r\n\tat org.openqa.selenium.grid.node.local.LocalNode.newSession(LocalNode.java:384)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor.startSession(LocalDistributor.java:645)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor.newSession(LocalDistributor.java:564)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.handleNewSessionRequest(LocalDistributor.java:818)\r\n\tat org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable.lambda$run$1(LocalDistributor.java:779)\r\n\tat java.base\u002fjava.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base\u002fjava.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base\u002fjava.lang.Thread.run(Thread.java:833)\r\n","exception.type": "org.openqa.selenium.SessionNotCreatedException","logger": "org.openqa.selenium.grid.distributor.local.LocalDistributor","request.payload": "[Capabilities {browserName: internet explorer, se:ieOptions: {ie.edgechromium: true}}]"}}
```

### Operating System

Windows 10, Windows 11

### Selenium version

Java 4.8.1

### What are the browser(s) and version(s) where you see this issue?

Edge 111.0.1661.51 (Official build) (64-bit)

### What are the browser driver(s) and version(s) where you see this issue?

IEDriverServer (32-bit) version 4.8.1

### Are you using Selenium Grid?

4.8.1

**Comment by titusfortner** [comment]
try starting the server with `--selenium-manager true` at the end.
If that doesn't work, use `--log-level "finer"` and provide the results in a gist.

**Comment by diemol** [comment]
This was fixed by https://github.com/SeleniumHQ/selenium/commit/ed0fe601775a35432e2492a5d9b0952eb9ce9ed2 and available in 4.8.2
